pkg/workloadmeta/types.go

```diff
@@ -50,14 +50,9 @@ type Source string
 
 // Defined Sources
 const (
```

Comment:
Not directly related to this PR, but would it make sense to have a source representing all sources? 
`const All Source = ""`